packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -97,7 +93,7 @@ export interface TableOptions {
    * Whether server-side encryption with an AWS managed customer master key is enabled.
    * @default - server-side encryption is enabled with an AWS owned customer master key
    */
-  readonly serverSideEncryption?: boolean;
+  readonly serverSideEncryption?: TableEncryption;
```

Comment:
We cannot incur breaking changes on a `stable` API (which this is). See my top-level comment for more information on how to deal with that.

Reply:
Thanks! I deprecated `serverSideEncryption` and introduced `encryption` prop. Do we also need to deprecate existing `grant` methods? As new `grant` methods are modified to support encrypt/decrypt and objectsKeyPattern

Reply:
No I reckon you simply want to update the existing `grant` methods to do the right thing.